### DISCUSSION TITLE
Compatibility of another RS485 module

### DISCUSSION THREAD

**Discussion by P8ntBal1551** [esphome-econet/esphome-econet#6] [Q&A]
Hi,

As digikey says that the K045 RS485 module is "obsolete or no longer manufactured", I looked for something equivalent on aliexpress. As I have never worked with RS485, I am sorta out of my depth here. [Would this module work as well?](https://www.aliexpress.com/item/4000987346232.html)

Thank you

**Comment by barndawgie** [comment]
I think you basically need two pieces: The R485 interface and an esp32 micro-controller. I _think_ that is just the interface and you would need to also buy the m5 ATOM itself.

The unit we linked includes both components in a little case, but I also see it marked as deprecated (though I recently bought two).

**Reply by kaijk** [reply]
Mouser has the K045s in stock for now.  https://www.mouser.com/ProductDetail/M5Stack/K045?qs=81r%252BiQLm7BQ2ho0A5VkoNw%3D%3D

Otherwise, order an A131 and a C008 from M5Stack themselves.
